Neo4j.Driver/Neo4j.Driver.Tests/StatementResultReaderExtensionsTests.cs

```diff
@@ -31,10 +31,10 @@ public class SingleAsyncMethod
             [Fact]
             public async Task ShouldReturnSingleRecord()
             {
-                var mock = new Mock<IStatementResultReader>();
+                var mock = new Mock<IStatementResultCursor>();
```

Comment:
Can we also change this file name from `StatementResultReaderExtensionsTests` to `StatementResultCursorExtensionsTests`? same for the extension class itself.